genki_signals/frontends/visualization.py

```diff
@@ -66,7 +66,8 @@ def __init__(self, video_key: str):
         self.widget = Image(format="jpeg")
 
     def update(self, data: DataBuffer):
-        value = data[self.video_key][..., -1].transpose(2, 1, 0)
+        transpose = (2, 1, 0) if data[self.video_key].ndim == 4 else (1, 0)
```

Comment:
is this so it also works with black/white picture mode?

Reply:
yes